ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -287,9 +293,14 @@ def context_check_botmessage(botmessage):
         Returns:
             True
         """
-        for word in botmessage.lower().strip().split():
-            if word in NAME_VARIATIONS:
-                return True
+        if self.language == HINDI_LANG:
```

Comment:
clean bot message and then check if `" " + name + " "` lies in string `" " + botmessage + " "`